test/e2e/push_test.go

```diff
@@ -116,6 +116,15 @@ var _ = Describe("Podman push", func() {
 		push := podmanTest.Podman([]string{"push", "-q", "--tls-verify=false", "--remove-signatures", ALPINE, "localhost:5000/my-alpine"})
 		push.WaitWithDefaultTimeout()
 		Expect(push).Should(Exit(0))
+		Expect(len(push.ErrorToString())).To(Equal(0))
+
+		push = podmanTest.Podman([]string{"push", "--tls-verify=false", "--remove-signatures", ALPINE, "localhost:5000/my-alpine"})
+		push.WaitWithDefaultTimeout()
+		Expect(push).Should(Exit(0))
+		Expect(push.ErrorToString()).To(ContainSubstring("Copying blob "))
+		Expect(push.ErrorToString()).To(ContainSubstring("Copying config "))
+		Expect(push.ErrorToString()).To(ContainSubstring("Writing manifest to image destination"))
+		Expect(push.ErrorToString()).To(ContainSubstring("Storing signatures"))
```

Comment:
also below here is a skip, we should not skip in the middle of a test, consider changing this to `if !remote{...}`